vibi-dpu/src/utils/gitops.rs

```diff
@@ -253,7 +254,9 @@ fn process_diff(filepath: &str, diff: &str, linemap: &mut HashMap<String, Vec<St
 	let mut limiterpos = Vec::new();
 	let delimitter = "@@";
 	for (idx, _) in diff.match_indices(delimitter) {
-		limiterpos.push(idx);
+		if has_deletions(&diff[idx..]) {
+            limiterpos.push(idx);
```

Comment:
can you please describe this?! I kind of have a hunch but still confused. Baaki I am approving the PR.

Reply:
checking if hunk has any deletions present in it